drivers/network/netio/netio.c

```diff
@@ -0,0 +1,860 @@
+/*
+ * PROJECT:     NetIO driver
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     A more convenient networking (TCP/IP, UDP/IP) kernel API (NT6+)
+ * COPYRIGHT:   Copyright 2023-2024 Johannes Thoma <johannes@johannesthoma.com>
+ */
+
+/* We are a NT6+ driver */
+#undef _WIN32_WINNT
+#define _WIN32_WINNT 0x600
+#undef WINVER
+#define WINVER 0x600
```

Comment:
If your NT kernel driver depends on "WIN32_WINNT" or "WINVER" defines, you can be 99.999999% sure you're doing something you must not do somewhere...

Reply:
Hi I found that if I apply some minor patches to the headers it compiles without forcing the NT version to 6. Will push a commit in a minute...

Reply:
Ok now after fixing the headers (a bit) it also compiles without changing WINVER.